modules/platforms/dotnet/Apache.Ignite.Core/Impl/Cache/Query/FieldsQueryCursor.cs

```diff
@@ -33,7 +34,7 @@ internal class FieldsQueryCursor<T> : PlatformQueryQursorBase<T>
         /// Constructor.
         /// </summary>
         /// <param name="target">Target.</param>
-        /// <param name="keepBinary">Keep poratble flag.</param>
+        /// <param name="keepBinary">Keep portable flag.</param>
```

Comment:
`portable` -> `binary`.

P.S. This is an ancient legacy of early Ignite days. `Binary` was called `Portable`, then we renamed, but this one line was skipped due to a typo.